create_mysql.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+
+var servers = require('./servers.json');
+var new_servers = [];
+
+const mysql_app = {
```

Comment:
instead of `mysql_app` change to `mysqlServer`?  let's change all variables to be `camelCase` instead.